docs/use-cases/deep-ssm-femur.md

```diff
@@ -0,0 +1,57 @@
+# DeepSSM: ShapeWorks Deep Learning on Femur Data
```

Comment:
I think this use case should be in use-cases/deep-learning, right ?

Reply:
@sheryjoe What do you think? Should we have a deep-learning folder inside the use-cases folder?